packages/swapper/src/thorchain-utils/getL1RateOrQuote.ts

```diff
@@ -131,7 +139,7 @@ export const getL1RateOrQuote = async <T extends ThorTradeRateOrQuote>(
   const recommendedMinimumCryptoBaseUnit = swapQuote.recommended_min_amount_in
     ? convertPrecision({
         value: swapQuote.recommended_min_amount_in,
-        inputExponent: nativePrecision,
+        inputExponent: sellAssetNativePrecision,
```

Comment:
Naming doesn't look correct - this seems to imply that this is the precision of the native asset for the sell asset - in reality, it's the native precision of the either the *fee* asset for the chain (i.e RUNE/CACAO precision), *or* MAYA right?

Reply:
Although I'm not sure I can come with something better here so meh!

Reply:
So it is intended on representing the precision of the asset in term of Thorchain/Mayachain base units. This amount to precision of 8 for everything except for CACAO which is precision 10. So I have been using "native" to imply the network base units if you will. I didn't want it to mean the actual precision on the asset as it may exist on it's own network however which certainly could be confused with the "native" notion.

I can add a comment at minimum for explanation, but there really isn't a great name outside of thormayaPrecision or something stupid like that lol

Reply:
00e5c6fb576c5758cfcb5c99a82108a98156efe1